src/containers/BooksForm.js

```diff
@@ -17,23 +21,28 @@ class BooksForm extends React.Component {
       title: '',
       category: '',
     };
-    this.updateInput = this.updateInput.bind(this);
-    this.updateSelect = this.updateSelect.bind(this);
+    this.handleChange = this.handleChange.bind(this);
```

Comment:
- [ ] **[OPTIONAL]** Since we have used arrow functions to define handlers, we can safely remove bindings these functions. Read more about this behavior [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/this#arrow_functions)